docker-compose.yml

```
@@ -0,0 +1,43 @@
+version: '3.8'
+services:
+  hashtopolis:
+    container_name: hashtopolis
+    build: .
+    restart: always
+    volumes:
+      - ./src/import/:/var/www/html/import/:Z
+      - ./src/files/:/var/www/html/files/:Z
+      - ./data/conf.php:/var/www/html/inc/conf.php:Z
```

The whole folder can be mounted without an issue, this make the conf.php persist and conf.php is already in gitignore.
```suggestion
      - ./src/inc/:/var/www/html/inc/:Z
```